CMakeLists.txt

```diff
@@ -208,9 +208,9 @@ if (NOT HCC_VERSION_STRING)
 endif()
 
 # Set HCC version string. The rule for version string is:
-# HCC_VERSION_MAJOR . HCC_VERSION_MINOR . HCC_VERSION_PATCH-KALMAR_SDK_COMIT-KALMAR_FRONTEND_COMMIT-KALMAR_BACKEND_COMMIT
+# HCC_VERSION_MAJOR . HCC_VERSION_MINOR . HCC_VERSION_PATCH-KALMAR_SDK_COMIT-LLVM_COMMIT
```

Comment:
This doesn't match the variable name used below

Reply:
Do you mean, at line #288? where we set "KALMAR_BACKEND" ?  I don't think that variable is used in the version string.